frappe/public/js/frappe/ui/filters/filter.js

```diff
@@ -542,13 +542,13 @@ frappe.ui.filter_utils = {
 			if (period_map[period]) {
 				period_map[period].forEach((p) => {
 					options.push({
-						label: __(`{0} {1}`, [period, p]),
+						label: __("{0} {1}", [period, p]),
```

Comment:
This does not make sense. This will just concatenate `period` and `p` without translating anything.